### ISSUE TITLE
where can I find the details about boxes normalization?

### ISSUE THREAD

**Issue by alpha-gradient** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#87]
when I use object detections I found the parameters of boxes are in range from 0 to 1,how can I get the ground truth of these boxes?
thanks for your attention
